corehq/apps/app_manager/views/apps.py

```diff
@@ -945,6 +949,19 @@ def pull_master_app(request, domain, app_id):
     return HttpResponseRedirect(reverse_util('app_settings', params={}, args=[domain, app_id]))
 
 
+@require_can_edit_apps
+def pull_missing_multimedia(request, domain, app_id):
```

Comment:
Can this and related methods move to `linked_domain` app or can we make a new app called `linked_apps`?

Reply:
I don't see why not

Reply:
@snopoke `linked_apps` or `linked_domain`?

Reply:
linked_domain

Reply:
https://github.com/dimagi/commcare-hq/pull/25010/commits/1ce375dc4af17100c6d94109ed366d2aacaa0743